### ISSUE TITLE
Warp book mod

### ISSUE THREAD

**Issue by CommandantCypher** [xJon/The-1.12.2-Pack#59]
**If your feature request is a mod suggestion:**
1. Has someone suggested this mod before: Yes
2. Why should the mod be added: The warp book mod on its own is very well done, and a simple mod, however, another notable reason is for the player pages. Currently, in the 1.12.2 pack, there are no good ways of tracking players, and one would be player pages. I play on the HaxysHideout PVP server, and there is no good way of finding people on it other than running around for hours finding nothing.

**Mod info & download page**
* Link to official page: https://www.curseforge.com/minecraft/mc-mods/warp-book

**Comment by xJon** [comment]
What would be the exact use of finding people on the server in this way? What's the difference between teleporting to their coordinates?
And also don't you think it's too overpowered of a way to teleport to other players, especially on a PvP server?

**Comment by CommandantCypher** [comment]
Essentially, using a player page, you can trick the player into binding it to themselves, waiting for them to get to their base and using it to teleport to them... Its a way to track people, or find their coordinates persay, as of now there is no other way of doing this in the pack, and playing on a PVP server without functioning tracking is kind of annoying seeing as its raid orientated and you can't find people to raid..

**Comment by CommandantCypher** [comment]
Another simple use for it is, you have a team-mate and you want to be able to instantly teleport to his location in case of an emergency you can have a player warp page on hand ready to help him..

**Comment by Furry** [comment]
I don't really like CommandantCypher's reasoning, but currently, there are massive teleportation limitations. Without a multi-block setup, there's a limit to how far you can teleport.

With player pages, you can teleport an infinite distance, which is much prefered, versus having to try to get closer to your destination so your teleporting tool has enough RF.

**Comment by CommandantCypher** [comment]
That is also another benefit, also, another thing to add on is that it is an original mod from the 1.7.10 pack making it even more authentic if added to the 1.12, however, I also suggest adding in a tracking item, for PVP where you can get a player's coordinates by clicking on them with an item, similar to the tracker from defense tech, however that one doesn't work at the moment.

**Comment by xJon** [comment]
@Furry I disagree, as the modpack has Whoosh installed.
@CommandantCypher What about the Wireless Tracker?

**Comment by CommandantCypher** [comment]
What @Furry is saying, is that Whoosh doesn't let you teleport infinitely, and has limitations as the original 1.7.10 pack had warp books, making travel so much easier than the Whoosh mod, if your whole goal is to keep things from 1.7.10, then removing the warp book mod is a bad call, as it was used by pretty much everyone.

As for the Wireless Tracker, it is clunky, easy to shake, and doesn't output coordinates. People also just know how to avoid it, if you are in a fight with someone, and before they teleport out, the tracker is removed if you have fire aspect on a sword, as it gets rid of it, the tracker is pretty much useless as its almost impossible to track anyone with it.

**Comment by xJon** [comment]
Whoosh is much more balanced than the Warp Book mod, and with the Holding enchantment + Flux Capacitors, it's basically infinite.
Regarding the tracking, I agree that especially with the lack of Witchery, more content can be included to enhance PvP, but I'm not sure whether Warp Books is the answer.

**Comment by CommandantCypher** [comment]
I am, It's an original mod, and it keeps people happy. I think it really helps with some things that are lacking in the sense that it provides what a vanilla server already would in the form of plugins (/home) except in a tangible object ingame. As for the PvP aspect, I wouldn't solely recommend warp books, however, it is a "Hotfix" in my eyes, as it adds in some way. At the moment there isn't any reasonable way to find people other than running around with a pneumatic helmet trying to find someone on a 100k x 100k map. 

To summarize: the warp book mod is not only good for raid servers, (Just an added bonus of the mod) but it is also just nice to have in general, all players seem to love it, and although it might look imbalanced, the later versions balance it where you need to make a potion to be able to set warps. Not only is it something people love, and helps temporarily solve the PvP problem, but it is also an origional mod, (giving it more reason to be in the pack) and it's a pleasent mod to have.

**Comment by xJon** [comment]
Regardless of the PvP aspect of player tracking, how is the WarpBook feature significantly better than Whoosh's features? The power requirements are not very high and insignificant. Do you see it as a *magic* alternative to teleportation over *tech*?

**Comment by CommandantCypher** [comment]
I see it 1. As a magic alternative to tech, and also I personally, (if presented with the choice) would choose a warp book over a creative whoosh tool, I just like the simplicity, and enjoy the menu. It is simple, and done well.

**Comment by CommandantCypher** [comment]
As of now, I have just tried the wireless tracker on a test server with a friend, it doesn't stick to the person, and it bugs into the floor. It hardly works, as of now, there is no good way of finding people's coordinates after interacting with them. Also, to comment on the pneumatic helm, it only shows you entities 16 blocks away, that is barely any range, and at that rate going through a map with 75k 75k it would take you weeks of non stop running around to even have a chance at finding anything... Its very unefficent and un-realistic.

**Comment by Furry** [comment]
@xJon Hate to comment in an old ticket, but the mod "Woosh" has insanely high power costs, and often teleporting, even with a completely maxed out teleporter, still cant because the demand for power is too high.

[Example](http://dragons.are-very.sexy/Y87GvFfP)

With Warpbooks we didn't have this limitation. We can teleport an infinite distance. Woosh is insanely underpowered, It's worse then mekanism's teleporting mechanic. 

@CommandantCypher there's always a way. I used to use a Digital Quarry to scan for blocks from certain tech mods as it flys on a funky locomotion flying machine. If it sees one, it would give me the cords it would tell me so I can TP to it. Im sure there's many ways to do it in this pack too, just have to think.

**Comment by xJon** [comment]
Written by one of the admins of a PvP server running the pack:
> Player pages were proven to be detrimental to the playerbase on 1.7.10
pvp servers thrive when there is new blood coming to the server, anyone who got stung by a player page scam to start would forever be unable to play due to the fact it is not 1 time use, and unless you were able to take it and all copies of it back from the other user, you would never be safe.
so players would just leave once they realized the severity of having another user with their player page.

And I agree. Additionally, Whoosh never used 800K RF in my use. If it did, this could be looked at as a separate issue/enhancement.
If you have any other mod suggestions to enhance the PvP experience in a balanced way, make sure to open a new issue.
